adapters/Unity/UnityAdapter/AdNetworkAdapterProxy/GADMUnityBannerNetworkAdapterProxy.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC.
+// Copyright 2021 Google LLC.
```

Comment:
@chlandacastro github shows that this file (and a few more .h files) is renamed when actually it's a new file, so not sure if I should have 2020 here. Please let me know if I need to change this.